parallel_gripper_controller/include/parallel_gripper_controller/parallel_gripper_action_controller_impl.hpp

```diff
@@ -237,27 +237,34 @@ void GripperActionController::check_for_success(
   {
     if (fabs(current_velocity) > params_.stall_velocity_threshold)
     {
-      last_movement_time_ = time;
+      last_movement_time_.try_set(time);
```

Comment:
Should we use `set` here? @christophfroehlich

Reply:
check_for_success is called inside the RT, so I was thinking it should be `try_set()`. Correct me if I'm wrong please.